copy/src/redux/types/index.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    SET_LOADING,
+    SET_QUESTIONS,
+    SET_USER_ANSWERS,
+    SET_SCORE,
+    SET_CURRENT_NUMBER,
+    SET_GAME_OVER
+} from '../actions'
+
+
+export enum Difficulty {
+    Easy = 'easy',
+    Medium = 'medium',
+    Hard = 'hard'
+};
+
+export interface AnswerObject {
+    question: string;
+    userAnswer: string;
+    isCorrect: boolean;
+    correctAnswer: string;
+};
+
+export interface GameState {
+    loading: boolean,
+    questions: QuestionState[],
+    userAnswers: AnswerObject[],
+    score: number,
+    currentNumber: number,
+    gameOver: boolean,
+}
+
+export interface Question {
+    category: string,
+    correct_answer: string,
```

Comment:
When naming attributes, you are mixing camelcase with underscore case - better not mix